docs/content/feature/grpc-proxy.md

```diff
@@ -14,6 +14,54 @@ fabio -proxy.addr ':1234;proto=grpc'
 
 As per the HTTP/2 spec, the host header is not required, so host matching is not supported for GRPC proxying.
 
+To route to a particular grpc service(s), set the 'dsthost'='name' in metadata, the grpc traffic will be routed to the service advertising `urlprefix-name/my.service/Method proto=grpc`.
+
+For example in Go grpc request:
+```
+	md := metadata.Pairs("dsthost", "testonly")
+	ctx := metadata.NewOutgoingContext(context.Background(), md)
+```
+To make RPC using the context with the metadata will try to access the service with tag
+```
+ urlprefix-testonly/my.service/Method proto=grpc
+```
+
+first, if no such service exits, fall back to the service with tag 
```

Comment:
Perhaps rephrase it as: <strikethrough>first,</strikethrough> If no service *exists*, fallback ...